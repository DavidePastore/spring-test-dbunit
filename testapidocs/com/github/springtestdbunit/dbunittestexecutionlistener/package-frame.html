<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_06) on Wed Jan 09 20:24:56 PST 2013 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>com.github.springtestdbunit.dbunittestexecutionlistener (Spring Test DBUnit 1.0.1 Test API)</title>
<meta name="date" content="2013-01-09">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/springtestdbunit/dbunittestexecutionlistener/package-summary.html" target="classFrame">com.github.springtestdbunit.dbunittestexecutionlistener</a></h1>
</body>
</html>
