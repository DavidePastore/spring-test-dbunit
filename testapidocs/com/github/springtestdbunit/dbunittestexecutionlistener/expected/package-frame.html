<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.6.0_26) on Sat Apr 21 14:47:47 PDT 2012 -->
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<TITLE>
com.github.springtestdbunit.dbunittestexecutionlistener.expected (Spring Test DBUnit 1.0.1-SNAPSHOT Test API)
</TITLE>

<META NAME="date" CONTENT="2012-04-21">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../../../../stylesheet.css" TITLE="Style">


</HEAD>

<BODY BGCOLOR="white">
<FONT size="+1" CLASS="FrameTitleFont">
<A HREF="../../../../../com/github/springtestdbunit/dbunittestexecutionlistener/expected/package-summary.html" target="classFrame">com.github.springtestdbunit.dbunittestexecutionlistener.expected</A></FONT>
<TABLE BORDER="0" WIDTH="100%" SUMMARY="">
<TR>
<TD NOWRAP><FONT size="+1" CLASS="FrameHeadingFont">
Classes</FONT>&nbsp;
<FONT CLASS="FrameItemFont">
<BR>
<A HREF="DoesNotSwallowExpectedFailureTest.html" title="class in com.github.springtestdbunit.dbunittestexecutionlistener.expected" target="classFrame">DoesNotSwallowExpectedFailureTest</A>
<BR>
<A HREF="ExpectedFailureOnClassTest.html" title="class in com.github.springtestdbunit.dbunittestexecutionlistener.expected" target="classFrame">ExpectedFailureOnClassTest</A>
<BR>
<A HREF="ExpectedFailureOnMethodTest.html" title="class in com.github.springtestdbunit.dbunittestexecutionlistener.expected" target="classFrame">ExpectedFailureOnMethodTest</A>
<BR>
<A HREF="ExpectedNonStrictOnClassTest.html" title="class in com.github.springtestdbunit.dbunittestexecutionlistener.expected" target="classFrame">ExpectedNonStrictOnClassTest</A>
<BR>
<A HREF="ExpectedNonStrictOnMethodTest.html" title="class in com.github.springtestdbunit.dbunittestexecutionlistener.expected" target="classFrame">ExpectedNonStrictOnMethodTest</A></FONT></TD>
</TR>
</TABLE>


</BODY>
</HTML>
